### PR TITLE
[#170840092] Apartment gem setup

### PR DESCRIPTION
**Pull request by Durgesh00**
Apartment configuration is done for the project!!

### CODE REVIEW

File: test/fixtures/organisations.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+# This model initially had no columns defined. If you add columns to the
+# model remove the '{}' from the fixture names and add the columns immediately
+# below each fixture, per the syntax in the comments below
+#
+one: {}
+# column: value
+#
+two: {}
+# column: value
```
Comment (smriti154) [inline test/fixtures/organisations.yml]: Remove this file

File: test/models/organisation_test.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class OrganisationTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (smriti154) [inline test/models/organisation_test.rb]: Remove Test folder

File: test/fixtures/organisations.yml (hunk 1 above)
Comment (Durgesh00) [reply test/fixtures/organisations.yml]: done

File: test/models/organisation_test.rb (hunk 2 above)
Comment (Durgesh00) [reply test/models/organisation_test.rb]: done

**Review by smriti154 — APPROVED** [review]

**Pull request merged**